Cargo.lock

```diff
@@ -873,8 +907,8 @@ dependencies = [
  "bytes",
  "headers-core",
  "http",
- "mime 0.3.16",
- "sha-1",
+ "mime",
+ "sha-1 0.8.2",
```

Comment:
Ditto

Reply:
can't fix... probably better than having two versions of rustls